File: sumOfMultiples/sumOfMultiples.js
function sumMul(n,m){
	//your idea here
	  if(n <= 0 || m <= 0)
		return "INVALID";
	  else
	  {
		var tmp = 0;
		var total = tmp;
		while(tmp < m)
		{
		  total += tmp;
		  tmp += n;
	//       console.log("tmp: ", tmp);
	//       console.log("total: ", total);
		}
		return total;
	  }
	}

File: sumOfMultiples/sumOfMultiples.cs
public class Kata
{
  public static int SumMul(int n, int m)
  {
    if(n <= 0 || m <= 0 || n > m || n == m)
    {    
      throw new System.ArgumentException("Parameter cannot be non positive integer or n > m");
    }
    else
    {
      int tmp = 0;
      int total = tmp;
      while(tmp <= m)
      {
        total += tmp;
        tmp += n;
      }
      return total;
    }    
  }
}